scripts/rollup/bundles.js

```diff
@@ -147,7 +157,7 @@ const bundles = [
     ],
     fbEntry: 'src/renderers/dom/shared/ReactDOMUnstableNativeDependenciesEntry',
     hasteName: 'ReactDOMUnstableNativeDependencies',
-    isRenderer: false,
+    moduleType: ISOMORPHIC,
```

Comment:
It's not "isomorphic" (we only call the main React package this).

Can we instead fix it to get `Object.assign` with `ReactInternals`?

Reply:
that’s what it was doing on this branch before the last commit. That’s why the file size was different...

Reply:
In that case the change was correct :-) As long as it works then we're good.

I didn't mean the change is necessarily wrong, I just meant that it is surprising and we should have a reasonable explanation.